src/components/BalancePage/BalancePage.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const BalancePage = () => (
+  <div className="balance__page">
+    <h1 className="balance__page_balance">Your current balance is: <span className="main__page_header-span">0,00 💲</span></h1>
+    <h3 className="balance__page_spent">Total spent: <span className="main__page_header-span">1034074028,00 💲</span></h3>
+    <h3 className="balance__page_received">Total received: <span className="main__page_header-span">1034074028,00 💲</span></h3>
+    <button className="balance__page_btn btn">
+      <Link to="/" className="link">
```

Comment:
I left you a comment regards the creating separate file with routes, and here is the best place to use `HOME` variable from this file instead of '/' ;)

Reply:
Fixed. Links are now like this <Link to={ERROR} className="link">